TimberAPI/CustomObjectSystem/CustomObjectRegistry.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TimberbornAPI.CustomObjectSystem
+{
+    public class CustomObjectRegistry : ICustomObjectRegistry
```

Comment:
Does the feature work infinetly: load game -> unload -> load -> etc..

Reply:
Hmm. It should but I can test it

Reply:
I think so